sycl/test/extensions/properties/mock_compile_time_properties.hpp

```diff
@@ -103,28 +103,28 @@ struct is_property_key_of<fir_key, syclObjectT> : std::true_type {};
 
 namespace detail {
 template <> struct PropertyToKind<bar_key> {
-  static constexpr PropKind Kind =
-      static_cast<enum PropKind>(PropKind::PropKindSize + 0);
+  static constexpr PropKind Kind = static_cast<enum PropKind>(
+      static_cast<std::uint32_t>(PropKind::PropKindSize) + 0);
```

Comment:
How did it fail? Could we do the same by using "0u"?

Reply:
It fails on Windows with a recent upstream patch with an error about ambiguity on the `+` operator. I can give `0u` a try.

Reply:
Sadly `0u` does not fix the issue seen, so I think the cast is the most stable solution.